external-docs/mkdocs.yml

```diff
@@ -25,6 +25,10 @@ nav:
           - Templates: guides/configuration/templates.md
           - Units: guides/configuration/units.md
           - Code Lists: guides/configuration/code-list-config.md
+      - Shape your data: guides/shape-data.md
```

Comment:
It might be useful to have a shape your data section with children here to keep things a bit tidier.

Reply:
Done